test/keycloak-connect-web-spec.js

```diff
@@ -285,6 +285,30 @@ test('Public client should work with slash in the end of auth-server-url', t =>
   });
 });
 
+test('App should be able to use cookie-store', t => {
+  t.plan(1);
+  var app = new NodeApp();
+  var client = admin.createClient(app.publicClient('appCookies'));
+
+  return client.then((installation) => {
+    app.build(installation, { cookies: true });
+    return page.get(app.port, '/cookie').then(() => {
+      page.login('alice', 'password').then(() => {
+        driver.navigate().refresh().then(() => {
+          page.events().getText().then(text => {
+            t.equal(text, 'Auth Success', 'User should be authenticated');
```

Comment:
@vmuzikar should we check here that output element contains stuff from `keycloak-token` cookie? Or the presence of `Auth Success` is enough for us and implies that the cookies were readable?

Reply:
Good point. I believe the `Auth Success` should be enough. If the `keycloak-token` cookie wouldn't contain a valid input, it would fail on JSON parsing here:
https://github.com/keycloak/keycloak-nodejs-connect/pull/263/files#diff-2c6e47101e422ddc2041c33fe87447ffabb5da93c296e0d89426b91cef5d75feR132

Reply:
I quickly tried this: 
```
console.log(JSON.parse(null))
```
And it works, but I am not sure what `req.cookies['keycloak-token']` returns when there is no `keycloak-token` cookie.

Reply:
It should return `undefined` if the cookie is missing (I accidentally experienced it myself when preparing the fix).

But to be on the safe side, I've added a small if:
https://github.com/keycloak/keycloak-nodejs-connect/blob/fd245b43359b7790319544ab230b221495a6e4e9/test/fixtures/node-console/index.js#L132